trainingRN/src/screens/todo/layout.js

```diff
@@ -12,8 +13,13 @@ function TodoLayout({ handleDelete, data, handleNew, setNewText }) {
         <View style={styles.taskBox} key={element.key}>
           <Text style={styles.task}>{element.task}</Text>
           <Button title="X" onPress={() => handleDelete(element.id)}/>
```

Comment:
I'd extract this function to an instance method, so we avoid recreating it on every render. What do you think ?

Reply:
I thought of doing that. But i need to pass the `element.id` from the map. So i need to create this fn inside of it. Do you think there is another way to do it?